src/main/java/org/entando/entando/aps/system/services/entity/model/EntityDto.java

```diff
@@ -73,6 +73,7 @@ public void fillEntity(IApsEntity prototype, ICategoryManager categoryManager, B
         prototype.setDescription(getDescription() == null ? prototype.getDescription() : getDescription());
         prototype.setMainGroup(getMainGroup() == null ? prototype.getMainGroup() : getMainGroup());
         if (null != this.getGroups()) {
+            prototype.getGroups().clear();
```

Comment:
Add a `setGroups` method and avoid the 2 step invocation? Minor optimization

Reply:
It seems wise, updated